### PR TITLE
README.md Cleanup

### PR DESCRIPTION
**Pull request by omgitsbillryan**
- Corrections for accuracy
- Moved all feature-specific info into separate .md files in `/docs/setup/`
- Make the README a bit shorter and easier to read

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,131 +40,37 @@ For an example, see `application.yml.example`
...
-Curl or browse to `http://localhost:3001/v0/sessions/new`; copy and paste the ID.me URL into your browser. Create your ID.me account if you have not already done so (**Note**: creating your account on the ID.me site is separate from the api.idmelabs.com sandbox) or sign in with your username and password.
-
-The token returned in the json response at the end of the login flow can be used as follows (You may wish to use Postman instead of curl to test within the browser):
-
-```
-curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyyja" localhost:3001/v0/sessions/current
-curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyyja" localhost:3001/v0/profile
-```
-
 ## Deployment Instructions
```
Comment (omgitsbillryan) [inline README.md]: I'm on the fence WRT removing "Deployment Instructions".  Seems to me deployment info belongs in the `devops` repo, and probably already exists in there.  @jkassemi thoughts?

**Comment by omgitsbillryan** [comment]
@lihanli - is the bit about `X-Key-Inflection` still accurate([here](https://github.com/department-of-veterans-affairs/vets-api/blame/master/README.md#L186))?  I question if it is worthy of our front-page readme.md

@markolson - is the foreman stuff still up to date? ([link](https://github.com/department-of-veterans-affairs/vets-api/blame/master/README.md#L148))

@robbiethegeek - do you have the contact email for reporting of security-related bugs?

**Comment by carlmjohnson** [comment]
`X-Key-Inflection` is still a thing. I think it's worth explaining because otherwise you'll be confused why curl and your browser give different key names.

**Comment by markolson** [comment]
@omgitsbillryan foreman is still good

File: README.md (hunk 1 above)
Comment (jkassemi) [inline README.md]: I think it's worth a pointer to the devops repo (https://github.com/department-of-veterans-affairs/devops#vets-api), but I like the instructions for determining the currently deployed version (frequently the first step in troubleshooting an issue). Details on the deployment pipeline are probably not much use here.

**Pull request merged**